### ISSUE TITLE
Is this what the training should look like ?

### ISSUE THREAD

**Issue by shahaamirbader** [ZikangZhou/QCNet#6]
Hi I have been training the model for two days now on 2 x V100 GPUs 32 GB. This is what the output at my terminal looks like. Can you comment on if the training is going in the right direction? I see some losses increasing here. Also how many epochs is the training suppose to last for?

![image](https://github.com/ZikangZhou/QCNet/assets/78397774/e56f2a4a-1062-4564-83bf-52245d3a856e)

**Comment by ZikangZhou** [comment]
The training looks good. In your screenshot, isn't the loss generally decreasing (some losses are negative)? You can monitor the training process using tensorboard or wandb.

The default max_epoch is set to 64, but you can adjust those training hyperparameters, as it seems that your batch size is current set to 8.

**Comment by HIT-five** [comment]
How long will the model training last if  I use the 8 NVIDIA 3090? thank you~

**Comment by ZikangZhou** [comment]
> How long will the model training last if I use the 8 NVIDIA 3090? thank you~

Two days I think.
